.github/actions/rust-build-release/tests/test_cross_install.py

```
@@ -444,6 +444,7 @@ def test_falls_back_to_cargo_when_runtime_unusable(
     main_module: ModuleType,
     cross_module: ModuleType,
     module_harness: HarnessFactory,
+    setup_manifest: Path,
```

Ruff (0.15.12)

[warning] Unused function argument: `setup_manifest`

(ARG001)